docs/references/astro/locals.mdx

```diff
@@ -15,8 +15,10 @@ You can use the `auth()` local to protect your pages. This local will return the
 
 ```astro filename="src/pages/protected.astro"
 ---
-if (!Astro.locals.auth().userId) {
-  return Astro.redirect('/login');
+const { userId, redirectToSignIn } = Astro.locals.auth();
+
+if (!userId) {
+  return redirectToSignIn();
```

Comment:
We added the `redirectToSignIn` helper in page-level so I figured it would be nice to showcase it here as well